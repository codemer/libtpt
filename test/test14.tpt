@set(vara, "Old Var A")\
@set(varb, "Old Var B")\
@set(varc, "Old Var C")\
Main Body
vara=${vara}
varb=${varb}
varc=${varc}
@macro(mymac1, vara, varb, varc)
{
@mymac2(vara, varb, varc)\
mymac1()
vara=${vara}
varb=${varb}
varc=${varc}
}
@macro(mymac2, vara, varb, varc)
{
mymac2()
varc=${varc}
varb=${varb}
vara=${vara}
}
@mymac1("New Var A", "New Var B", "New Var C")\
Main Body
vara=${vara}
varb=${varb}
varc=${varc}

