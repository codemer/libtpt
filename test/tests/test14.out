Main Body
vara=Old Var A
varb=Old Var B
varc=Old Var C
mymac2()
varc=New Var C
varb=New Var B
vara=New Var A
mymac1()
vara=New Var A
varb=New Var B
varc=New Var C
Main Body
vara=Old Var A
varb=Old Var B
varc=Old Var C

